facilitator/src/lib.rs

```diff
@@ -17,6 +17,10 @@ pub use workflow::{workflow_main, WorkflowArgs};
 
 pub const DATE_FORMAT: &str = "%Y/%m/%d/%H/%M";
 
+/// Identity represents a cloud identity: Either an AWS IAM ARN (i.e. "arn:...")
+/// or a GCP ServiceAccount (i.e. "foo@bar.com").
+pub type Identity = String;
```

Comment:
The type alias is adequate for now, but it'd be neat if this was an enum akin to `enum StoragePath` in `config.rs`. Come to think of it that would be a more natural place for this defintion.